VTTiny.Plugin.Discord/DiscordBot.cs

```diff
@@ -0,0 +1,95 @@
+﻿using DSharpPlus;
+using DSharpPlus.Entities;
+using DSharpPlus.SlashCommands;
+using DSharpPlus.VoiceNext;
+using Microsoft.Extensions.DependencyInjection;
+using VTTiny.Plugin.Discord.Commands;
+using VTTiny.Plugin.Discord.Services;
+
+public class DiscordBot
```

Comment:
This class needs to be in the VTTiny.Plugin.Discord namespace.

Reply:
in dotnet 6+ the main entrypoint is the Csproj name, but i'll explicitly put it there